packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1070,6 +1087,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.deltaManager.on("op", this.onOp);
         }
 
+        this.logger.sendTelemetryEvent({
+            eventName: "ContainerLoadStats",
+            containerLoadDataStoreCount: this.dataStores.containerLoadDataStoreCount,
```

Comment:
You're renaming this from `dataStoreCount`. Intentional?  Better to leave it as-is and avoid churn, unless the name is misleading and you've given some thought to who may be expecting the old field name and need help transitioning to the new one.

Reply:
yes it's intentional because Navin mentioned the old name was misleading. I have not thought about who may be expecting the old name🤔🤔